kubectl-rook-ceph.sh

```diff
@@ -16,33 +16,269 @@
 
 set -eEuo pipefail
 
-usages() {
-  cat <<EOF
-Description:
-  'kubectl rook-ceph ceph' provides common management and troubleshooting tools for Ceph.
+####################################################################################################
+# HELPER FUNCTIONS
+####################################################################################################
 
-Commands:
-  ceph - run any 'ceph' CLI command as given
-    args  - any 'ceph' CLI argument or flag
+function print_usage () {
+  echo ""
+  echo "DESCRIPTION"
+  echo "kubectl rook-ceph provides common management and troubleshooting tools for Ceph."
+  echo "USAGE"
+  echo "  kubectl rook-ceph <main args> <command> <command args>"
+  echo "MAIN ARGS"
+  echo "  -v, --verbose               : output verbose info"
+  echo "  -h, --help                  : output help text"
+  echo "  -n, --namespace='rook-ceph' : the namespace of the CephCluster"
+  echo "COMMANDS"
+  echo "  help           : output help text"
+  echo "  ceph <args>    : call a 'ceph' CLI command with arbitrary args"
+  echo "  status         : show status of the CephCluster and its resources"
+  echo "    -l, --long     : show long status"
+  echo "  operator <subcommand>..."
+  echo "    restart      : restart the Rook-Ceph operator"
```

Comment:
wrong indent?

Reply:
this is a subcommand of `operator` that's why it is inside the operator.